packages/lesswrong/lib/utils/jsonUtils.ts

```diff
@@ -4,6 +4,9 @@
  * but does *not* wrap it in quotes or escape quotes.
  */
 export const toEmbeddableJson = (serializable: unknown): string => {
-  return JSON.stringify(serializable)
-    .replace(/<\//g, "<\\/")
+  const json = JSON.stringify(serializable);
+  if (json === undefined) {
+    return 'undefined';
+  }
+  return json.replace(/<\//g, "<\\/");
```

Comment:
If this change is necessary to prevent issues then I suspect that there's some other problem somewhere; nothing calling this function should be passing in `undefined`.